mmf/models/pythia.py

```diff
@@ -81,8 +82,9 @@ def _init_feature_encoders(self, attr):
 
         for feat_encoder in feat_encoders_list_config:
             feat_encoder_config = copy.deepcopy(feat_encoder)
-            feat_encoder_config.params.model_data_dir = self.config.model_data_dir
-            feat_encoder_config.params.in_dim = feature_dim
+            with omegaconf.open_dict(feat_encoder_config):
+                feat_encoder_config.params.model_data_dir = self.config.model_data_dir
+                feat_encoder_config.params.in_dim = feature_dim
```

Comment:
We should make this change for `movie_mcan` model as well.

Reply:
Done. Thanks for pointing out.